railties/lib/rails/generators/generated_attribute.rb

```diff
@@ -65,11 +65,13 @@ def parse(column_definition)
         end
 
         def dangerous_name?(name)
-          ActiveRecord::Base.dangerous_attribute_method?(name)
+          defined?(ActiveRecord::Base) &&
```

Comment:
This will never allow any name if `ActiveRecord::Base` is not defined.

```suggestion
          !defined?(ActiveRecord::Base) ||
```

Reply:
Ah my bad probably I'm wrong.